lib/scsi_defs.h

```diff
@@ -80,5 +87,60 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  // SCSI Reference Manual Table 148 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  enum class SelectReport : uint8_t {
+    kRestrictedMethods = 0x0,
+    kWellKnown = 0x1,
+    kAllLogical = 0x2
+  };
+
+  // SCSI Reference Manual Table 147 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  typedef struct ReportLunsCommand {
```

Comment:
Do we want a constructor for this? I briefly mentioned this in the chat that having an explicit ctor might help with readability and ease of use later on. Also the ctor can check whether the data is actually providing the correct opcode. Open to thoughts!

Reply:
Good point - if we are going to try using constructors, all structs should have one as a standard.